src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
It should be the block class first and then the modifier. You can't use a modifier without a block. Fix in all places

Reply:
fixed